css/src/atomics/border.scss

```diff
@@ -0,0 +1,59 @@
+$border-positions: (('top', 'right', 'bottom', 'left'));
+$user-positions: (('top', $user-right, 'bottom', $user-left));
+
+.has-border {
+	border: $border-width solid $border !important;
+}
+
+.has-border-none {
+	border: 0 !important;
+
+	&-tablet {
+		@include tablet {
+			border: 0 !important;
+		}
+	}
+
+	&-desktop {
+		@include desktop {
+			border: 0 !important;
+		}
+	}
+
+	&-widescreen {
+		@include widescreen {
+			border: 0 !important;
+		}
+	}
+}
+
+// Rules for each side
+@for $i from 1 through 4 {
+	$border-position: nth($border-positions, $i);
+	$user-position: nth($user-positions, $i);
+
+	.has-border-#{$border-position} {
+		border-#{$user-position}: $border-width solid $border !important;
+	}
+
+	.has-border-#{$border-position}-none {
+		border: 0 !important;
+	}
+}
+
+.has-border-high-contrast {
+	border: $border-width solid $border-white-high-contrast;
+}
+
+.has-border-high-contrast-hover:hover {
+	border: $border-width solid $border-yellow-high-contrast;
+}
+
+// Border radius
+.has-border-radius {
+	border-radius: $border-radius !important;
+}
+
+.has-border-radius-large {
+	border-radius: $border-radius-large !important;
+}
```

Comment:
Thoughts on this general scheme?

```txt

// mainstays
border
border-none
border-left
border-right
border-bottom
border-top

// screen specific
border-tablet
etc ...
border-none-tablet
etc ...
border-left-desktop
etc ...

// modifiers to the above
border-color-primary
border-width-l
border-radius
border-radius-l
```

Reply:
Agreed, wanted to confirm if we want to get rid of `has-` prefix! I'll update

Reply:
I was also debating if we need to do screen-specific rules for each side, like `border-left, `border-right`, etc. But since you mentioned it  as well - I'll add those as well

Reply:
I think we really need to come up with some limiting strategy for screensize based helpers. We may want to consider adding configuration options for it.

Reply:
Updated, please lmk if the new approach works! I tried to keep as few rows as possible and avoid any duplications